[2023-08-30T18:56:37.303+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:37.312+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T18:56:37.320+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T18:56:37.322+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T18:56:37.339+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-29 00:00:00+00:00
[2023-08-30T18:56:37.350+0000] {standard_task_runner.py:57} INFO - Started process 502 to run task
[2023-08-30T18:56:37.356+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmppqkiw3v0']
[2023-08-30T18:56:37.360+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask install_required_packages
[2023-08-30T18:56:37.420+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:37.541+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [running]> on host 5e7fbf8b7901
[2023-08-30T18:56:37.578+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:37.699+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T18:56:37.711+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T18:56:37.717+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T18:56:37.730+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T18:56:39.291+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T18:56:40.038+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T18:56:40.102+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T18:56:40.117+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T18:56:40.123+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T18:56:40.125+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T18:56:40.127+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T18:56:40.128+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T18:56:43.827+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy
[2023-08-30T18:56:43.884+0000] {subprocess.py:93} INFO - Successfully installed sodapy-2.2.0
[2023-08-30T18:56:44.275+0000] {subprocess.py:93} INFO - 
[2023-08-30T18:56:44.277+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T18:56:44.279+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T18:56:44.554+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T18:56:44.607+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230829T000000, start_date=20230830T185637, end_date=20230830T185644
[2023-08-30T18:56:44.668+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T18:56:44.694+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:19:23.803+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:19:23.817+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:19:23.827+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:19:23.828+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:19:23.844+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-29 00:00:00+00:00
[2023-08-30T19:19:23.849+0000] {standard_task_runner.py:57} INFO - Started process 218 to run task
[2023-08-30T19:19:23.855+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpyoqedv_z']
[2023-08-30T19:19:23.859+0000] {standard_task_runner.py:85} INFO - Job 6: Subtask install_required_packages
[2023-08-30T19:19:23.918+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:19:24.026+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [running]> on host 884d86529f65
[2023-08-30T19:19:24.061+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:19:24.184+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T19:19:24.187+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:19:24.191+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T19:19:24.211+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:19:28.617+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:19:29.494+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:19:29.596+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:19:29.606+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:19:29.617+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:19:29.620+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:19:29.622+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:19:29.623+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:19:57.613+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy
[2023-08-30T19:19:57.676+0000] {subprocess.py:93} INFO - Successfully installed sodapy-2.2.0
[2023-08-30T19:19:57.891+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:19:57.893+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:19:57.894+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:19:58.138+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:19:58.199+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230829T000000, start_date=20230830T191923, end_date=20230830T191958
[2023-08-30T19:19:58.267+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:19:58.310+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:31:06.368+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:06.383+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:31:06.397+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:31:06.398+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:31:06.419+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-29 00:00:00+00:00
[2023-08-30T19:31:06.425+0000] {standard_task_runner.py:57} INFO - Started process 219 to run task
[2023-08-30T19:31:06.429+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpbn_jed76']
[2023-08-30T19:31:06.433+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask install_required_packages
[2023-08-30T19:31:06.492+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:06.625+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [running]> on host f6bcf103ff79
[2023-08-30T19:31:06.653+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:06.792+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T19:31:06.797+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:31:06.800+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T19:31:06.819+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:31:08.987+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:31:11.134+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:31:11.268+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:31:11.284+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:31:11.293+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:31:11.298+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:31:11.300+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:31:11.302+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:31:32.958+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy
[2023-08-30T19:31:33.110+0000] {subprocess.py:93} INFO - Successfully installed sodapy-2.2.0
[2023-08-30T19:31:33.251+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:31:33.252+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:31:33.253+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:31:33.512+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:31:33.612+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230829T000000, start_date=20230830T193106, end_date=20230830T193133
[2023-08-30T19:31:33.676+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:31:33.717+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:41:18.492+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:18.510+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:41:18.526+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:41:18.529+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:41:18.545+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-29 00:00:00+00:00
[2023-08-30T19:41:18.551+0000] {standard_task_runner.py:57} INFO - Started process 277 to run task
[2023-08-30T19:41:18.566+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmptug5oav9']
[2023-08-30T19:41:18.573+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask install_required_packages
[2023-08-30T19:41:18.649+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:18.746+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [running]> on host 81eeb4b7c8a9
[2023-08-30T19:41:18.774+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:18.878+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T19:41:18.896+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:41:18.899+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T19:41:18.918+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:41:24.308+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:41:33.472+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:41:33.829+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:41:33.850+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:41:33.860+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:41:33.862+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:41:33.864+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:41:33.865+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:41:49.426+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy
[2023-08-30T19:41:49.549+0000] {subprocess.py:93} INFO - Successfully installed sodapy-2.2.0
[2023-08-30T19:41:49.557+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:41:49.559+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:41:49.560+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:41:49.867+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:41:49.896+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230829T000000, start_date=20230830T194118, end_date=20230830T194149
[2023-08-30T19:41:49.920+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:41:49.945+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:50:05.877+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:05.890+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:50:05.912+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:50:05.914+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:50:05.945+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-29 00:00:00+00:00
[2023-08-30T19:50:05.961+0000] {standard_task_runner.py:57} INFO - Started process 245 to run task
[2023-08-30T19:50:05.972+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpgxkro0dk']
[2023-08-30T19:50:05.979+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask install_required_packages
[2023-08-30T19:50:10.852+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:10.992+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [running]> on host 206ecb1dc8df
[2023-08-30T19:50:11.076+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:11.255+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T19:50:11.291+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:50:11.301+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy geoalchemy2']
[2023-08-30T19:50:11.363+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:50:25.501+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:50:30.715+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:50:30.867+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:50:31.234+0000] {subprocess.py:93} INFO - Collecting geoalchemy2
[2023-08-30T19:50:31.255+0000] {subprocess.py:93} INFO -   Downloading GeoAlchemy2-0.14.1-py3-none-any.whl (54 kB)
[2023-08-30T19:50:31.301+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 55.0/55.0 kB 1.5 MB/s eta 0:00:00
[2023-08-30T19:50:31.312+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:50:31.319+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (23.1)
[2023-08-30T19:50:31.320+0000] {subprocess.py:93} INFO - Requirement already satisfied: SQLAlchemy>=1.4 in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (1.4.49)
[2023-08-30T19:50:31.336+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:50:31.339+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:50:31.340+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:50:31.341+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:50:31.433+0000] {subprocess.py:93} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from SQLAlchemy>=1.4->geoalchemy2) (2.0.2)
[2023-08-30T19:50:48.944+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy, geoalchemy2
[2023-08-30T19:50:49.119+0000] {subprocess.py:93} INFO - Successfully installed geoalchemy2-0.14.1 sodapy-2.2.0
[2023-08-30T19:50:49.578+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:50:49.579+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:50:49.580+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:50:49.844+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:50:49.926+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230829T000000, start_date=20230830T195005, end_date=20230830T195049
[2023-08-30T19:50:50.003+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:50:50.054+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:54:01.065+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:01.083+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:54:01.096+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:54:01.098+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:54:01.123+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-29 00:00:00+00:00
[2023-08-30T19:54:01.158+0000] {standard_task_runner.py:57} INFO - Started process 206 to run task
[2023-08-30T19:54:01.172+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpzc83ymgi']
[2023-08-30T19:54:01.193+0000] {standard_task_runner.py:85} INFO - Job 7: Subtask install_required_packages
[2023-08-30T19:54:01.249+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:01.361+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [running]> on host 86f9e97d6c70
[2023-08-30T19:54:01.390+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:01.492+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T19:54:01.497+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:54:01.499+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy geoalchemy2']
[2023-08-30T19:54:01.516+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:54:04.584+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:54:05.176+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:54:05.281+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:54:05.335+0000] {subprocess.py:93} INFO - Collecting geoalchemy2
[2023-08-30T19:54:05.650+0000] {subprocess.py:93} INFO -   Downloading GeoAlchemy2-0.14.1-py3-none-any.whl (54 kB)
[2023-08-30T19:54:05.675+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 55.0/55.0 kB 2.4 MB/s eta 0:00:00
[2023-08-30T19:54:05.678+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:54:05.682+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (23.1)
[2023-08-30T19:54:05.682+0000] {subprocess.py:93} INFO - Requirement already satisfied: SQLAlchemy>=1.4 in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (1.4.49)
[2023-08-30T19:54:05.693+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:54:05.695+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:54:05.697+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:54:05.698+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:54:05.758+0000] {subprocess.py:93} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from SQLAlchemy>=1.4->geoalchemy2) (2.0.2)
[2023-08-30T19:54:23.813+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy, geoalchemy2
[2023-08-30T19:54:23.916+0000] {subprocess.py:93} INFO - Successfully installed geoalchemy2-0.14.1 sodapy-2.2.0
[2023-08-30T19:54:23.925+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:54:23.927+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:54:23.928+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:54:24.183+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:54:24.250+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230829T000000, start_date=20230830T195401, end_date=20230830T195424
[2023-08-30T19:54:24.310+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:54:24.392+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:05:28.921+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:28.935+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T20:05:28.957+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T20:05:28.958+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:05:28.981+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-29 00:00:00+00:00
[2023-08-30T20:05:28.987+0000] {standard_task_runner.py:57} INFO - Started process 420 to run task
[2023-08-30T20:05:28.994+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp3dbh80qr']
[2023-08-30T20:05:28.999+0000] {standard_task_runner.py:85} INFO - Job 13: Subtask install_required_packages
[2023-08-30T20:05:33.811+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:37.416+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [running]> on host 3938d7abd9ff
[2023-08-30T20:05:37.443+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:37.554+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T20:05:37.557+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:05:37.559+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy geoalchemy2']
[2023-08-30T20:05:37.581+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:05:41.593+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T20:05:41.836+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T20:05:41.837+0000] {subprocess.py:93} INFO - Requirement already satisfied: geoalchemy2 in /home/***/.local/lib/python3.8/site-packages (0.14.1)
[2023-08-30T20:05:41.838+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T20:05:41.841+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (23.1)
[2023-08-30T20:05:41.843+0000] {subprocess.py:93} INFO - Requirement already satisfied: SQLAlchemy>=1.4 in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (1.4.49)
[2023-08-30T20:05:41.847+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T20:05:41.848+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T20:05:41.849+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T20:05:41.850+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T20:05:41.893+0000] {subprocess.py:93} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from SQLAlchemy>=1.4->geoalchemy2) (2.0.2)
[2023-08-30T20:05:45.558+0000] {subprocess.py:93} INFO - 
[2023-08-30T20:05:45.563+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T20:05:45.565+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T20:05:45.826+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:05:45.906+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230829T000000, start_date=20230830T200528, end_date=20230830T200545
[2023-08-30T20:05:45.976+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:05:45.999+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:33:32.763+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:32.782+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T20:33:32.793+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T20:33:32.794+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:33:32.809+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-29 00:00:00+00:00
[2023-08-30T20:33:32.814+0000] {standard_task_runner.py:57} INFO - Started process 245 to run task
[2023-08-30T20:33:32.825+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpuwif0t02']
[2023-08-30T20:33:32.829+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask install_required_packages
[2023-08-30T20:33:32.847+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:32.907+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [running]> on host 961f38e8363e
[2023-08-30T20:33:32.936+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:33.071+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T20:33:33.091+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:33:33.093+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy geoalchemy2']
[2023-08-30T20:33:33.123+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:33:40.404+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T20:33:40.784+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T20:33:40.788+0000] {subprocess.py:93} INFO - Requirement already satisfied: geoalchemy2 in /home/***/.local/lib/python3.8/site-packages (0.14.1)
[2023-08-30T20:33:40.790+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T20:33:40.795+0000] {subprocess.py:93} INFO - Requirement already satisfied: SQLAlchemy>=1.4 in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (1.4.49)
[2023-08-30T20:33:40.796+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (23.1)
[2023-08-30T20:33:40.801+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T20:33:40.802+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T20:33:40.803+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T20:33:40.804+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T20:33:40.850+0000] {subprocess.py:93} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from SQLAlchemy>=1.4->geoalchemy2) (2.0.2)
[2023-08-30T20:33:47.448+0000] {subprocess.py:93} INFO - 
[2023-08-30T20:33:47.453+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T20:33:47.455+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T20:33:47.572+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:33:47.597+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230829T000000, start_date=20230830T203332, end_date=20230830T203347
[2023-08-30T20:33:47.644+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:33:47.697+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:48:15.000+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:15.010+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T20:48:15.022+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T20:48:15.023+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:48:15.035+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-29 00:00:00+00:00
[2023-08-30T20:48:15.040+0000] {standard_task_runner.py:57} INFO - Started process 293 to run task
[2023-08-30T20:48:15.048+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp_7fx2ty2']
[2023-08-30T20:48:15.052+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask install_required_packages
[2023-08-30T20:48:15.079+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:15.184+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [running]> on host 8f66711a2898
[2023-08-30T20:48:15.197+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:15.305+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T20:48:15.307+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:48:15.309+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T20:48:15.326+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:48:16.851+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T20:48:17.167+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T20:48:17.169+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T20:48:17.180+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T20:48:17.181+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T20:48:17.183+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T20:48:17.184+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T20:48:21.096+0000] {subprocess.py:93} INFO - 
[2023-08-30T20:48:21.098+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T20:48:21.100+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T20:48:21.333+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:48:21.378+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230829T000000, start_date=20230830T204815, end_date=20230830T204821
[2023-08-30T20:48:21.429+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:48:21.454+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T21:07:24.715+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:24.729+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T21:07:24.744+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T21:07:24.746+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T21:07:24.760+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-29 00:00:00+00:00
[2023-08-30T21:07:24.770+0000] {standard_task_runner.py:57} INFO - Started process 267 to run task
[2023-08-30T21:07:24.781+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpubucqwm2']
[2023-08-30T21:07:24.789+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask install_required_packages
[2023-08-30T21:07:24.833+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:24.911+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [running]> on host cbc745b7550b
[2023-08-30T21:07:24.939+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:25.067+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T21:07:25.071+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T21:07:25.073+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T21:07:25.098+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T21:07:26.989+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T21:07:27.184+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T21:07:27.186+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T21:07:27.191+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T21:07:27.194+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T21:07:27.195+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T21:07:27.196+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T21:07:33.187+0000] {subprocess.py:93} INFO - 
[2023-08-30T21:07:33.191+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T21:07:33.193+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T21:07:33.416+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T21:07:33.462+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230829T000000, start_date=20230830T210724, end_date=20230830T210733
[2023-08-30T21:07:33.516+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T21:07:33.554+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T21:16:37.371+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:16:37.384+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T21:16:37.392+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T21:16:37.394+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T21:16:37.407+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-29 00:00:00+00:00
[2023-08-30T21:16:37.412+0000] {standard_task_runner.py:57} INFO - Started process 308 to run task
[2023-08-30T21:16:37.420+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp49i2nyuk']
[2023-08-30T21:16:37.424+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask install_required_packages
[2023-08-30T21:16:37.453+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:16:37.538+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-29T00:00:00+00:00 [running]> on host 5b9f14bf20e0
[2023-08-30T21:16:37.553+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:16:37.674+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T21:16:37.679+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T21:16:37.681+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T21:16:37.700+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T21:16:48.826+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T21:16:55.857+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T21:16:55.862+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T21:16:55.875+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T21:16:55.876+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T21:16:55.879+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T21:16:55.881+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T21:17:01.281+0000] {subprocess.py:93} INFO - 
[2023-08-30T21:17:01.283+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T21:17:01.284+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T21:17:01.431+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T21:17:01.460+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230829T000000, start_date=20230830T211637, end_date=20230830T211701
[2023-08-30T21:17:01.517+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T21:17:01.541+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
